pkg/chain/ethereum/ethereum.go

```diff
@@ -22,15 +22,19 @@ import (
 var logger = log.Logger("keep-chain-ethereum")
 
 // ThresholdRelay converts from ethereumChain to beacon.ChainInterface.
-func (ec *ethereumChain) ThresholdRelay() relaychain.Interface {
+func (ec *ethereumChain) ThresholdRelay() relayChain.Interface {
 	return ec
 }
 
 func (ec *ethereumChain) GetKeys() (*operator.PrivateKey, *operator.PublicKey) {
 	return operator.EthereumKeyToOperatorKey(ec.accountKey)
 }
 
-func (ec *ethereumChain) GetConfig() *relaychain.Config {
+func (ec *ethereumChain) Signing() chain.Signing {
+	return ethutil.NewSigner(ec.accountKey.PrivateKey)
+}
```

Comment:
At first I didn't like the naming that `NewSigner` returns `Signing` (but not `Signer`). But `ethutil.NewSigner` returns a struct that implements `chain.Signing`. I think it's fine.

Reply:
Nah, it should be `Signer` instead of `Signing`.